cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,86 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignUpPage from '../support/pages/signUp.pageObject';
+import ArticlePage from '../support/pages/article.pageObject';
+import faker from 'faker';
+import { confirmationMessage } from '../plugins/alertMessages';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('User', () => {
+  let user;
+  let followButtonSelector;
+  let followButtonText;
+  let unfollowButtonSelector;
+  let unfollowButtonText;
+  const signUpPage = new SignUpPage();
+
   before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      cy.visit('/#/register');
```

Comment:
You are visiting the URL '/#/register' directly. Try to navigate to the page using the UI whenever possible.

Reply:
It's starting point of all process